test/NuGet.Clients.Tests/NuGet.CommandLine.Test/RestoreNETCoreTest.cs

```diff
@@ -9795,7 +9795,7 @@ public async Task RestoreNetCore_CPVMProject_DowngradedByCentralDirectDependency
         ///  P will be accepted (because its parent B is Accepted)
         ///  S will be accepted (because its parent O 300 is Accepted)
         /// </summary>
-        [Fact]
+        [Fact(Skip = "Depends on cpvm transitive pinning")]
```

Comment:
We usually include a link to a tracking issue either as the skip reason, or as a comment next to the skipped test. This makes it easier to figure out when a test can be unskipped, or maybe deleted permanently.

Reply:
Updated